server/src/test/java/game/world/WorldChunkTest.java

```diff
@@ -65,5 +66,20 @@ void testAddASnake() {
                 .orElseThrow();
     }
 
+    @Test
+    void testFindIntersectingChunks() {
+        var worldChunkSize = world.getConfig().chunks.size;
+        var boundingBox = new BoundingBox(Vector.ORIGIN, 1, 1);
+        var worldChunks = world.chunks.findIntersectingChunks(boundingBox);
+        var originWorldChunk = worldChunks.stream().findAny().orElseThrow();
```

Comment:
This seems weird. I assume `originWorldChunk` is supposed to be the chunk intersecting with the origin. But relying on the order of elements in a set seems wrong. I recommend to use this instead: `var originWorldChunk = world.chunks.findChunk(Vector.ORIGIN);`